### DISCUSSION TITLE
2023 · 谈谈职业规划 | 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#249] [General]
# 2023 · 谈谈职业规划 | 愧怍的小站

又到了年底写年终总结的时候了，说实话今年感觉没什么内容可写。上半年我已经写了 叙一名转专业+休学的大学生经历，而下半年我忙于学校课程 + 课程重修，要说编程上真没什么好总结的。

https://kuizuo.cn/blog/2023-year-end-summary/

**Comment by huanfe1** [comment]
博主的 RSS 链接好像访问不了了

**Reply by kuizuo** [reply]
感谢反馈！之前将所有博客数据都放置在路径 /blog 下，忘记将 feed 更改过去了。

**Comment by MongoRolls** [comment]
博主，很感谢你，最近看你的博客，自己也搭建了博客，提一个小反馈：你的网站存在手机页面上会有主页会显示错误的bug哦

**Reply by kuizuo** [reply]
或许是浏览器兼容的问题？能否提供你所使用的手机浏览器以及版本，并附上相关截图以便排查🙏

**Reply by MongoRolls** [reply]
浏览器是夸克最新的，个人展示和自我介绍这一部分有问题
![](https://mongorolls-images.oss-cn-shenzhen.aliyuncs.com/img/11798332eb323b6a73c521c5b5cd63b.jpg)
![](https://mongorolls-images.oss-cn-shenzhen.aliyuncs.com/img/04e57d7633aacbdfb3339706d798690.jpg)

**Comment by kkkh** [comment]
非常优秀的博客

**Comment by Nil7x** [comment]
老哥是 XIT 的吗😄

**Reply by kuizuo** [reply]
我超，这你都知道。垃圾的不能再垃圾的，好在是毕业了，逃离这这个是非之地，狗都不去的那种。

**Comment by young-trigold** [comment]
从推过来的😂 因为对博主的经历比较好奇 整体浏览下感觉博主是喜欢冒险自由的 项目经历非常丰富 我也是前端 不过是在bat里面

**Comment by AaronConlon** [comment]
可以加一个友链吗
